src/styl/responsive/mobile.styl

```diff
@@ -0,0 +1,37 @@
+// HEADER
+
+.header-title
+  position: fixed
+  z-index: 100
+  text-align: center
+
+#header-title-venice, #header-title-post
+  left: 10%
+  width: 80%
+
+#header-title-venice
+  top: 0px
+  padding-top: $margin-basic
+
+#header-title-post
+  bottom: 0px
+  padding-bottom: $margin-basic
+
+#header-title-about, #header-title-contact
+  top: 50vh
```

Comment:
this doesn't vertically center these items. 

![screen shot 2017-10-30 at 1 17 48 pm](https://user-images.githubusercontent.com/7727722/32190925-5cda5572-bd7d-11e7-8c03-99df47c89648.png)

you could `top: calc(50vh - ($itemHeight / 2))`, or `height: 100vh; text-align: center`, or use flex